packages/api/src/controllers/experiment/attestation.ts

```diff
@@ -0,0 +1,103 @@
+import { Router } from "express";
+import { v4 as uuid } from "uuid";
+import { db } from "../../store";
+import { validatePost } from "../../middleware";
+import { ethers } from "ethers";
+import { makeNextHREF, toStringValues } from "../helpers";
+import _ from "lodash";
+
+const app = Router();
+
+const DOMAIN = {
+  name: "Verifiable Video",
+  version: "1",
+};
+
+app.post("/", validatePost("attestation"), async (req, res) => {
+  const { primaryType, domain, message, signature } = req.body;
+
+  if (primaryType !== "VideoAttestation") {
+    return res.status(400).json({ errors: ["invalid primaryType"] });
+  }
+  if (!_.isEqual(domain, DOMAIN)) {
+    console.log(domain);
```

Comment:
Should either remove or clean this up if we're looking to provide debugging context

Reply:
Yeah, sorry. leftover. Updated in e397ed1c1d13bf078b73616f70f7e51c19a0670f